### ISSUE TITLE
Login to Nomad Oasis does not work.

### ISSUE THREAD

**Issue by weidtn** [FAU-LAP/NOMAD-CAMELS#101]
When trying to log in to our Nomad Oasis, first of all, the "Get the token!' button does  is hardcoded to only work on windows with os.startfile.

It is not clear which kind of token we need. But when creating and entering an "App token" (and not an "auth token" on the API dashboard), the window disappears and then opens again. I have to insert the token again, and then I get the following error:

```
  File "C:\Users\exp4-PT1-096\micromamba\envs\camels\lib\site-packages\nomad_camels\MainApp_v2.py", line 412, in login_logout_nomad
    self.show_nomad_upload()
  File "C:\Users\exp4-PT1-096\micromamba\envs\camels\lib\site-packages\nomad_camels\MainApp_v2.py", line 441, in show_nomad_upload
    uploads = nomad_communication.get_user_upload_names(self)
  File "C:\Users\exp4-PT1-096\micromamba\envs\camels\lib\site-packages\nomad_camels\nomad_integration\nomad_communication.py", line 143, in get_user_upload_names
    uploads = get_user_uploads(parent)
  File "C:\Users\exp4-PT1-096\micromamba\envs\camels\lib\site-packages\nomad_camels\nomad_integration\nomad_communication.py", line 126, in get_user_uploads
    return response.json()["data"]
  File "C:\Users\exp4-PT1-096\micromamba\envs\camels\lib\site-packages\requests\models.py", line 978, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
```

After closing that error message, I appear to be logged in, but have to log in again when trying to load samples or upload results.

**Comment by weidtn** [comment]
> When trying to log in to our Nomad Oasis, first of all, the "Get the token!' button does is hardcoded to only work on windows with os.startfile.

I wrote a fix for that: #102 

But this does not fix the rest.

**Comment by Jo-Leh** [comment]
Thanks for pointing this out and providing the fix! #102 is merged into development now. We plan to release the next version later this week.

Unfortunately I could not reproduce the RequestsJSONDecodeError you encountered. I added a some logging in the development branch to better understand what is going on there. If you find the time, it would be great to test this.

**Comment by weidtn** [comment]
Okay, that's weird. I am pretty sure I found the solution to that, added the comment and closed the issue. Well here it is:

By testing out different versions of nomad-oasis links, I found out that I can log in with `.../nomad-oasis/gui`. This is not correct, but still works somehow. Your Login procedure needs the link ending in `.../nomad-oasis/api/v1`.

So I just used the wrong link to log in, which then broke the rest of the program. 

Maybe you should mention somewhere that the needed link is the one ending in `.../nomad-oasis/api/v1`.
